pkg/apiserver/controller/NodeController.go

```diff
@@ -258,12 +258,12 @@ func (n *NodeController) UpdateStorageNodeDisk(ctx *gin.Context) {
 
 	// 获取path中的devicePath
 	devicePath := ctx.Param("devicePath")
-	fmt.Printf("devicePath = %v\n", devicePath)
+	log.Infof("devicePath = %v", devicePath)
 
 	var drb hwameistorapi.DiskReqBody
 	err := ctx.ShouldBind(&drb)
 	if err != nil {
-		fmt.Errorf("Unmarshal err = %v\n", err)
+		log.Infof("Unmarshal err = %v", err)
```

Comment:
log level should be `Error`, so you should use `log.WithError` or `log.Error` method